README.md

```diff
@@ -1,41 +1,41 @@
-# Astronomer Deploy Action
-Custom Github Action to create CI/CD pipelines that deploys Airflow code to Astro Airflow Deployments during a CI/CD workflow. See [Astronomer's CI/CD documentation](https://docs.astronomer.io/astro/ci-cd) page for more information on creating CI/CD pipelines for Astro.
+# Deploy Action
+You use Deploy Action to create CI/CD pipelines that automate the deployment of code changes to a Deployment. For more information about Astro CI/CD workflows, see [Automate code deploys with CI/CD](https://docs.astronomer.io/astro/ci-cd).
 
-This action should be used with a Github Actions workflow that runs when code is merged into `main` (or equivalent) branch. Works with DAG-Only Deploys enabled or disabled. Enabling Dag-Only Deploys will skip an image deploy if only files in `/dags` folder change. More information can be found in Astronomer's [DAG-only Deploys documentation](https://docs.astronomer.io/astro/deploy-code#deploy-dags-only).
+Deploy Action should only be used in a GitHub Actions workflow that runs when code is merged into a `main` or an equivalent branch. You can use the Deploy Action with DAG-only deploys activated or deactivated. When DAG-only deploys are activated, an image deploy is not completed when only the files in the `/dags` folder change. For more information about DAG-only deploys, see [Deploy DAGs only](https://docs.astronomer.io/astro/deploy-code#deploy-dags-only).
 
-This action will execute the following steps:
-1. Checkout your current repo
-2. Determine if only DAG code changed
-2. Build your Astro Project into an image if files outside the `/dags` folder changed
-3. Parse or Pytest your DAG code (optional)
-4. Push your image to Astro's image Registry if files outside the `/dags` folder changed
-5. Deploy your image to your Astro Deployment if files outside the `/dags` folder changed
-6. Deploy only your `dags/` folder to your Astronomer Deployment if only files inside the `/dags` folder changed
+The following is the workflow for Deploy Action:
+- Your current repository is checked out
+- It is determined if only DAG code changed
+- Your Astro project is built into an image when you change files outside the `/dags` folder
+- Optional. Your DAG code is parsed or tested with Pytest
+- Your image is pushed to the Astro image registry if you changed files outside the `/dags` folder
+- Your image is deployed to your Deployment if you changed files outside the `/dags` folder
+- Your `dags/` folder is deployed to your Deployment if you changed files inside the `/dags` folder
```

Comment:
```suggestion
- Check out your repository.
- Check whether your commit only changed DAG code.
- Optional. Test DAG code with pytest.
- If the commit included only changes to DAG code, push the change to Astro without building a new project image.
- If the change included changes to project configurations, rebuild your project image and deploy it to Astro.
```

Reply:
Removed the passive language here, as I think we can be explicit that the action does all of this stuff.